devtools/gradle/src/functionalTest/java/io/quarkus/gradle/FastJarFormatWorksTest.java

```diff
@@ -0,0 +1,155 @@
+package io.quarkus.gradle;
+
+import io.quarkus.test.devmode.util.DevModeTestUtils;
+import io.quarkus.utilities.JavaBinFinder;
+import org.apache.commons.io.FileUtils;
+import org.gradle.testkit.runner.GradleRunner;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Predicate;
+import java.util.regex.Pattern;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+import static org.awaitility.Awaitility.await;
+
+
+public class FastJarFormatWorksTest extends QuarkusGradleTestBase{
+
+
+    @Test
+    public void testFastJarFormatWorks() throws IOException, URISyntaxException {
+
+        final File projectDir= getProjectDir("test-that-fast-jar-format-works-for-gradle");
+
+        GradleRunner.create()
+                .forwardOutput()
+                .withPluginClasspath()
+                .withArguments(arguments("package", "DskipTests","-Dquarkus.package.type=fast-jar"))
+                .withProjectDir(projectDir)
+                .build();
+
+//        await().atMost(1, TimeUnit.MINUTES).until(() -> result.getProcess() != null && !result.getProcess().isAlive());
+//        assertThat(running.log()).containsIgnoringCase("BUILD SUCCESS");
+//        running.stop();
```

Comment:
Please remove this if it's not going to be used.

Reply:
I wasn't able to replicate this part from the Maven test, do we actually need it here for testing the time limit?

Reply:
Nope, we don't need it because it's handled automatically by Gradle

Reply:
Okay